helm/charts/kratos/templates/deployment.yaml

```diff
@@ -93,8 +93,15 @@ spec:
           image: {{ include "kratos.image" . }}
           imagePullPolicy: {{ include "kratos.imagePullPolicy" . }}
           command:
+          {{ if not (empty .Values.deployment.overrideArgs.command) }}
```

Comment:
The check is redundant, you should be able to use just `if .Values.deployment.overrideArgs.command`